analyses/bigquery_shim/bigquery_shim/dashboard.py

```diff
@@ -146,11 +146,12 @@ def convert_bigquery_results(f):
     # instead, we explicitly add the properties we know we care about at the end of this routine
     additional_properties = json.loads(d.pop("additional_properties") or "{}")
     newdict = {}
+
     for k, v in d.items():
-        pieces = list(map(revert, k.split("__")))
-        # print(pieces)
+        pieces = list(map(revert, k.split("__")))  # list of values such as clientId, buildId
+
         if len(pieces) == 1:
-            newdict[pieces[0]] = v
+            newdict[pieces[0]] = v if v else {}
```

Comment:
Another option would be:

```suggestion
            newdict[pieces[0]] = v or {}
```

This syntax is a bit more compact and avoids referring to `v` twice. But it's probably less obvious than the `v if v else {}` construction. I'm leaning towards keeping this the way you've expressed it here.

Reply:
oh to be honest I did not realise `v or {}` is valid syntax. Apparently in some languages this is called `elvis operator`([source](https://stackoverflow.com/questions/4978738/is-there-a-python-equivalent-of-the-c-sharp-null-coalescing-operator)) which I did not know.

I kind of like the `or` approach it does feel a bit cleaner...

Reply:
ok I guess PR got merged in the meantime, I'll remember or for value setting for the future though. Thanks @jklukas